lldb/unittests/Core/ProgressReportTest.cpp

```diff
@@ -0,0 +1,105 @@
+#include "Plugins/Platform/MacOSX/PlatformMacOSX.h"
```

Comment:
Don't forget to include the license header in this file

Reply:
Will do! I told myself I'd add it at the end then I fully forgot to do that 🙃